packages/wrangler/src/containers/deploy.ts

```diff
@@ -79,14 +77,21 @@ function createApplicationToModifyApplication(
 ): ModifyApplicationRequestBody {
 	return {
 		configuration: req.configuration,
-		instances: req.max_instances !== undefined ? 0 : req.instances,
+		// deprecated in favour of max_instances
+		instances: 0,
```

Comment:
`instances` seems to be optional, could it just be removed here (and in the other occurrences of this in this file)?